### DISCUSSION TITLE
:sh output doesn't disappear

### DISCUSSION THREAD

**Discussion by kendfss** [helix-editor/helix#12283] [Q&A] [answered]
Sorry if this' been covered, I searched but didn't find any relevant results.  
Is there a way to make the output of a `:sh ...` usage disappear when you've finished reading it without closing the entire editor?  

example  
If I run `:sh echo test` the word test shows up and follows the cursor around. For larger outputs this can make it hard to read what you're editing. Is there a builtin way out of this problem or just keep on using other terminal/multiplexer windows for this?

**Comment by the-mikedavis** [comment, marked as answer]
You can use the Escape key or Ctrl+c to close the popup

**Reply by kendfss** [reply]
Silly me. Thanks!
